### PR TITLE
Add bulk size argument on Symfony bundle

### PR DESCRIPTION
**Pull request by Korbeil**

### CODE REVIEW

**Review by lyrixx — APPROVED** [review]
Perfect 👍🏼

File: src/Bridge/Symfony/Resources/config/services.php [hunk 1]
```diff
@@ -57,7 +57,7 @@
             ->args([
                 '$client' => service(Client::class),
                 '$serializer' => service('serializer'),
-                '$bulkMaxSize' => 100,
+                '$bulkMaxSize' => abstract_arg('bulk size'),
```
Comment (damienalexandre) [inline src/Bridge/Symfony/Resources/config/services.php:60]: **❔ question:** ‏Is it normal there is no underscore in the name here? 'bulk size'

Comment (jdreesen) [reply src/Bridge/Symfony/Resources/config/services.php:60]: It is, because it's only a description, not a name or so.

File: README.md [hunk 2]
```diff
@@ -252,7 +252,10 @@ elastically:
                 transport:           '@JoliCode\Elastically\Transport\HttpClientTransport'
 
             # Path to the mapping directory (in YAML)
-            mapping_directory: '%kernel.project_dir%/config/elasticsearch'
+            mapping_directory:       '%kernel.project_dir%/config/elasticsearch'
+
```
Comment (lyrixx) [inline README.md:256]: Please please please, configure your IDE to automatically remove trailing white space

**Pull request merged**